### DISCUSSION TITLE
Release Names

### DISCUSSION THREAD

**Discussion by bjlittle** [pletzer/mint#30] [Q&A] [answered]
I'm loving your `mint` release names i.e., `aerest`, `threo`, `fower` and `fifta`. Are they Māori names?

What do they mean and do you have a list of future release names?

i.e., if I was to do a release next what would I call it?

**Comment by pletzer** [comment, marked as answer]
Hi @bjlittle It's old Anglo-Saxon, so first, second, third, etc. It's a sure way not to ever run out of names :)
